launch_testing/launch_testing/__init__.py

```diff
@@ -53,19 +54,19 @@ def on_stdout_lines(self, lines):
                 continue
             self.stdout_data.write(line + b'\n')
 
-            # Are we ready to quit?
-            if self.regex_match:
-                self.matched = re.match(self.expected_lines[0], line)
-            else:
-                output_lines = self.stdout_data.getvalue().splitlines()
-                self.matched = output_lines == self.expected_lines
-            if self.matched:
-                # We have enough output to compare; shut down my child
-                for td in self.launch_descriptor.task_descriptors:
-                    if td.name == self.name:
-                        td.terminate()
-                        self.killed = True
-                        return
+        # Are we ready to quit?
```

Comment:
I see.

Perhaps if this repository had been documented better, I would understand the nature of `on_stdout_lines`. I could not find any information about how `on_stdout_lines` behaves in my perusal of the source code of `launch`.